spark/hbase-spark/src/main/scala/org/apache/hadoop/hbase/spark/datasources/HBaseTableScanRDD.scala

```diff
@@ -110,16 +110,20 @@ class HBaseTableScanRDD(relation: HBaseRelation,
       hbaseContext: HBaseContext): Iterator[Result] = {
     g.grouped(relation.bulkGetSize).flatMap{ x =>
       val gets = new ArrayList[Get](x.size)
+      val pointsSet = new mutable.HashSet[String]()
```

Comment:
the parameter name sounds confused..
could it be `rowkeySet`?

Reply:
Make sense.